src/global.json

```diff
@@ -1,4 +1,12 @@
 {
+  "sdk": {
+    "version": "7.0.100-rc.2.22477.23",
+    "allowPrerelease": true,
+    "rollForward": "latestPatch"
+  },
+  "tools": {
+    "dotnet": "7.0.100-rc.2.22477.23"
```

Comment:
This will need to be changed in other locations as well (in the CI dotnet setup). Still, it should not be needed. Is this for a local change ?

Reply:
I updated CI to use whatever we have global.json, so `global.json` is the central place that tells us the .NET SDK version.